ehcache-impl/src/test/java/org/ehcache/impl/internal/loaderwriter/writebehind/WriteBehindProviderFactoryTest.java

```diff
@@ -44,6 +45,10 @@
  */
 public class WriteBehindProviderFactoryTest {
 
+  public static <T> Consumer<T> NOOP() {
```

Comment:
Don't default to no-ops. You're weakening the test. If you don't expect this to be called then assert here... if you do then assert that this is called, or assert on it's effects somehow.